handbook/engineering/roles.md

```diff
@@ -15,6 +15,28 @@ Sofware Engineers are individual contributors who build software to satisfy the
 - Help your teammates by empathetically [reviewing code](code_reviews.md) and [RFCs](../communication/rfcs/index.md).
 - Interview candidates and suggest improvements to our hiring process.
 
+## Project Lead
+
+Project Leads are Software Engineers that lead a specific project team to achieve its goals. This role is an additional set of responsibilities that any engineer may take on once they demonstrate the necessary skills.
+
+While Project Leads need to have strong technical skills, they are not necessarily the most technically experienced engineer on the team. People and organizational skills are as important as the technical chops, so that Project Leads can effectively coordinate and align with others.
+
+Some teams with more experienced members may need less guidance in certain areas and may even share responsibilities of this role across the team.
+
+### Responsibilities
+
+- Directly contribute code a minimum of roughly 50% of their time.
```

Comment:
IC work spans more than just code. Should we represent this here?

```suggestion
- Directly contribute through code, RFCs or PR reviews for a minimum of roughly 50% of their time.
```

Reply:
This was specifically about coding. RFCs would be about "organizing and delegating work" and PR reviews about "reviewing work" which is captured in another line item.

Reply:
Then I think I would prefer grouping all aspects of IC work under a single bullet point (and possibly assigning a higher "percentage of time spent"?). All aspects of IC work are equally important, yet the balance of time spent on each of them  can vary significantly. The important point to get across here is "most of the project's lead time is still spent on IC work".